bundles/core/org.eclipse.smarthome.core.library/src/main/java/org/eclipse/smarthome/core/library/items/LocationItem.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.smarthome.core.library.items;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.smarthome.core.items.GenericItem;
+import org.eclipse.smarthome.core.library.CoreItemFactory;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.PointType;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.eclipse.smarthome.core.types.State;
+import org.eclipse.smarthome.core.types.UnDefType;
+
+/**
+ * A LocationItem can be used to store GPS related informations, addresses...
+ * This is useful for location awareness related functions
+ *
+ * @author Gaël L'hopital
+ *
+ */
+public class LocationItem extends GenericItem {
+
+    private static List<Class<? extends State>> acceptedDataTypes = new ArrayList<Class<? extends State>>();
+    private static List<Class<? extends Command>> acceptedCommandTypes = new ArrayList<Class<? extends Command>>();
+
+    static {
+        acceptedDataTypes.add(PointType.class);
+        acceptedDataTypes.add(UnDefType.class);
+
+        acceptedCommandTypes.add(RefreshType.class);
```

Comment:
Should we possibly also accept PointType as a command? E.g. commanding a robot to move to a certain spot?